cpu/samd21/Kconfig

```diff
@@ -4,5 +4,35 @@
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 #
+config CPU_SERIES_SAMD21
+    bool
+    select CPU_FAMILY_SAM0
+    select HAS_CPU_SAMD21
+    select HAS_PUF_SRAM
+
+## Common CPU symbols
+if CPU_SERIES_SAMD21
+
+config CPU
```

Comment:
since this is being redefined in Kconfig, maybe I would just stick to the CPU_xxx symbols